static/app/utils/performance/histogram/spanCountHistogramQuery.tsx

```diff
@@ -39,21 +37,24 @@ function getHistogramRequestPayload(props: RequestProps) {
     precision,
     dataFilter,
   };
-  const additionalApiPayload = omit(eventView.getEventsAPIPayload(location), [
-    'sort',
-    'per_page',
-    'cursor',
-  ]);
-  const apiPayload = {...baseApiPayload, ...additionalApiPayload};
-  return apiPayload;
+
+  const {
+    sort: _s,
+    per_page: _p,
+    cursor: _c,
+    ...additionalApiPayload
+  } = eventView.getEventsAPIPayload(location);
+
+  return {...baseApiPayload, ...additionalApiPayload};
```

Comment:
If you use Object.assign you can avoid copying the base object, and just write on top of it.